SDKCLI/Sources/SDKCLI/Models/Version.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+
+struct Version {
```

Comment:
given now we have spm

is it possible to reuse https://developer.apple.com/documentation/packagedescription/version?

Reply:
Great suggestion, will look into using this!